api/reports.py

```diff
@@ -172,6 +172,17 @@ def get_average_violations(camera_id):
     }
 
 
+@reports_api.get("/{camera_id}/no_face_mask_detections")
+def get_no_face_mask_detections(camera_id):
+    validate_existence(camera_id)
+    total_faces_detected, no_face_mask_percentage = reports.no_face_mask_detections(camera_id)
+    return {
+        'total_faces_detected': int(total_faces_detected),
+        'no_face_mask_percentage': no_face_mask_percentage,
```

Comment:
I think we should use "number" instead of "no" or  "face_mask_usage_percentage"

Reply:
I sent the porcentage because is the indicator displayed in the frontend.
But yes, we can send the number and calculate the percentage in the frontend.

Reply:
Ohhh wait  I undestand that "no" means "no_facemask" For some weird reason I thought it meant number -_-

This makes sense